src/App.js

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
+import { Route, Routes } from "react-router-dom";
+import { BeersList } from './components/BeersList';
+import { Nav } from './views/nav';
+import { Home } from './views/Home';
+import { BeerDetail } from './views/BeerDetail';
+import { RamdonBeer } from './views/RamdonBeer';
+import { NewBeer } from './views/NewBeer';
 
 function App() {
   return (
     <div className="App">
+
+    <Nav/>
+    <Routes>
+    <Route path="/" element={<Home />} />
+    <Route path="beers" element={<BeersList />} />
+    <Route path="beers/:beerId" element={<BeerDetail />} />
+    <Route path='/random-beer' element={<RamdonBeer />}/>
+    <Route path='/new-beer' element={<NewBeer />}/>
+    </Routes>
```

Comment:
aquí te falta un fallback para cuando el usuario navegue a una ruta que no existe, que entonces le redireccione a la home:
`<Route path='*' element={<Navigate to='/' />} />`